tests/components/select/common.py

```diff
@@ -0,0 +1,25 @@
+"""Common helpers for select entity component tests."""
+
+from homeassistant.components.select import SelectEntity
+
+from tests.common import MockEntity
+
+
+class MockSelectEntity(MockEntity, SelectEntity):
+    """Mock Select class."""
+
+    _attr_current_option = None
```

Comment:
```suggestion
```

Reply:
The `_attr_current_option` must be retained, `_handle` will fall back to the `current_option` of the SelectEntity if no value is set. `_attr_current_option` has no default value, which leads to test failure as the property does not exists. We can set a default value when we instantiate MockSelectEntity, then `_attr_current_option` can be removed.